test/unit/interfaces/hapi.ts

```diff
@@ -174,4 +178,101 @@ describe('Hapi interface', () => {
       fakeServer.emit(EVENT, {response: {isBoom: true}}, {continue() {}});
     });
   });
+  describe('Hapi17', () => {
+    const errorsSent: ErrorMessage[] = [];
+    // the only method in the client that should be used is `sendError`
+    const fakeClient = {
+      sendError:
+          (errorMessage: ErrorMessage,
+           userCb?: (
+               err: Error|null, response: http.ServerResponse|null, body: {}) =>
+               void) => {
+            errorsSent.push(errorMessage);
+          }
+    } as {} as RequestHandler;
+
+    // the configuration should be not be needed to send errors correctly
+    const plugin = hapiInterface(fakeClient, {} as Configuration);
+
+    afterEach(() => {
+      errorsSent.length = 0;
+    });
+
+    it(`Plugin should have name and version properties`, () => {
```

Comment:
Nit. No need for this string to be a template literal.

Reply:
Fixed